### ISSUE TITLE
[Bug] vuetify-nuxt-module NuxtApp type interfering with Nuxt's NuxtApp type

### ISSUE THREAD

**Issue by mtdvlpr** [userquin/vuetify-nuxt-module#145]
Since I updated `vuetify-nuxt-module` from 0.6.0 to 0.6.5 I have issues with my plugins. Somehow `vuetify-nuxt-module` is overwriting the return type of UseNuxtApp() so that it only contains $vuetify and not my own provided plugins. See https://github.com/mtdvlpr/reproduction/tree/vuetify for a reproduction.

**Comment by userquin** [comment]
I Will revert last commit tmr about module augmentation

**Comment by mtdvlpr** [comment]
@userquin thanks! BTW, you're probably already aware, but Vuetify has finally released v3.4.0 so this module needs to update the references from prior lab components.

**Comment by mtdvlpr** [comment]
I'm getting the following error without specifying any labComponents in the config:
```
Failed to resolve import "vuetify/labs/VDatePicker" from "..\..\..\..\..\..\@nuxt-vuetify-configuration\vuetify-configuration". Does the file exist?
```
